README.md

```diff
@@ -1,275 +1,18 @@
-[![CircleCI](https://circleci.com/gh/compound-finance/compound-protocol.svg?style=svg&circle-token=5ed19932325c559a06f71f87d69012aedd2cf3fb)](https://circleci.com/gh/compound-finance/compound-protocol) [![codecov](https://codecov.io/gh/compound-finance/compound-protocol/branch/master/graph/badge.svg?token=q4UvsvVzOX)](https://codecov.io/gh/compound-finance/compound-protocol)
-
-Compound Protocol
+Anchor Protocol
```

Comment:
Would be nice to add something like the PR description here.

Reply:
done. Please resolve if ok